serverless/index-serverless-project-settings.asciidoc

```diff
@@ -1,8 +1,10 @@
 [[project-and-management-settings]]
 == Project and management settings
 
-include::./pages/project-and-management-settings.asciidoc[leveloffset=+2]
 
+
+include::./pages/project-and-management-settings.asciidoc[leveloffset=+2]
+include::./index-serverless-devtools.asciidoc[leveloffset=+2]
 include::./pages/project-settings.asciidoc[leveloffset=+2]
```

Comment:
We could flatten most of what's here or organize it thematically rather than have a whole lot of features listed randomly like in the UI.
For example, custom roles could go with the maybe soon former "manage your org" if it turns into "manage users and roles"